src/gui/src/InstrumentEditor/InstrumentEditor.cpp

```diff
@@ -697,6 +723,20 @@ void InstrumentEditor::rotaryChanged(Rotary *ref)
 		if ( ref == m_pRandomPitchRotary ){
 			m_pInstrument->set_random_pitch_factor( fVal );
 		}
+		else if ( ref == m_pPitchCoarseRotary ) {
+			float newPitch = m_pPitchFineRotary->getValue() / 100.0 + (int)fVal ;
```

Comment:
Exactly, as in the layer editor

Reply:
Cast is definitely pointless here, `round(fVal)` should be consistent with the widget

Reply:
Also, since there is not a separate LCD for finePitch as in the layer tab, there's no need for multiplying the PitchFine widget value times 100 while setting, and then dividing by 100 after getting